src/openforms/utils/context_processors.py

```diff
@@ -22,8 +24,52 @@ def settings(request):
     if hasattr(django_settings, "SENTRY_CONFIG"):
         context.update(dsn=django_settings.SENTRY_CONFIG.get("public_dsn", ""))
 
-    context["settings"]["MAX_FILE_UPLOAD_SIZE"] = filesizeformat(
+    context["settings"]["MAX_FILE_UPLOAD_SIZE"] = filesizeformat_integers(
         django_settings.MAX_FILE_UPLOAD_SIZE
     )
 
     return context
+
+
+def filesizeformat_integers(bytes_):
+    """
+    Format the value like a 'human-readable' file size but with integer numbers (i.e. 13 KB, 4 MB,
+    102 bytes, etc.).
+
+    This is adapted from django/template/defaultfilters.py:838 (filesizeformat)
+    """
+    try:
```

Comment:
I'm not really a fan of all the code duplication for a single-character change. Maybe we can be a bit more pragmatic and do something like:

```py
django_formatted = filesizeformat(bytes_)
decimal_suffix = formats.number_format(0, 1)[1:]
return django_formatted.replace(decimal_suffix, "", 1)
```

also not particularly elegant, but more focused on the problem at hand.